website/content/docs/ui/custom-messages/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+page_title: Custom messages
+description: |-
+  Custom messages allow operators to configure system-wide messages.
+---
+
+# Custom Messages
+
+-> You must have **Vault 1.16.0 or higher** installed.
+
+Custom messages allow operators to configure system-wide messages and for vault users to view these messages.
+
+# Configure a Custom Message
+
+-> You must be a Vault operator or have capabilities to create a message.
+
+1. To create a message, navigate to the Settings section in the sidebar called **Custom Messages**.
+   ![custom messages sidebar link](/img/custom-messages-sidebar.png)
+2. On the **custom messages** page, each tab indicates where you would like the custom message to be created.
+3. Select the tab you want to create the message and click the **Create message** button in the toolbar.
+   ![custom messages create message button](/img/custom-messages-create.png)
+4. After clicking the **Create message** button, you are brought to the Create message form.
+5. On the **Create message** form:
+
+   ~> Note: When creating a message on a parent namespace, the child namespaces will inherit those messages.
+
+   - Select the location in which you want to create your message.
+   - Select the type of message you want to create.
+   - Fill in the message details like title, message, as well as an optional link that you might want to include at the end of your message.
+   - Select the message start time. The current message start time will default to the following day at 12am your local timezone.
+   - Select a message end time. By default messages will be never expire, however, there is also an option to set a specific expiration date.
+   - You may preview the message by clicking **Preview**.
+   - Click **Create message** to create the message. Now, the message will appear on the specified location at the specified date and time.
+
+# Edit a Custom Message
+
+-> You must be a Vault operator or have capabilities to create or edit a message.
+
+There are two ways to navigate to the edit custom message screen:
+
+- Navigate to the main screen of Custom messages. Find the custom message you want to edit and click the three dots on the right - it will prompt a dropdown with an option to select **Edit**.
+  ![custom messages edit message button on main screen](/img/custom-messages-edit-option-1.png)
+- Click the message you want to delete on the Custom message main screen and there should be a button called **Edit message**.
+  ![custom messages edit message button on message details screen](/img/custom-messages-edit-option-2.png)
+
+  After clicking **Edit message** in either of the options, fill in the information that you would like to edit for the custom message.
```

Comment:
Are there any fields that *can't* be edited on a message either before or after it has been published/released?

If the answer is "no", then I'd suggest deleting this sentence.

If the answer is "yes", then I'd rewrite this to address what information can/can't be edited and when it becomes immutable.

Reply:
Thanks for the suggestion! I will delete this sentence since there aren't any fields that can't be edited.